### PR TITLE
[CI] [doc] fix the dead link of resnet and mobilenet

### PR DESCRIPTION
**Pull request by shaoyuyoung**
Here are some dead links of resnet in the TVM doc.

one of these dead links has been caught in #17017 and a PR is submitted #17015

This PR fixes other dead links of resnet in tvm.

CC @leandron. When you do something about  cache file in S3, maybe you need to pay attention to this part :)

### CODE REVIEW

**Comment by yongwww** [comment]
@tvm-bot  rerun

**Comment by shaoyuyoung** [comment]
Well, what went wrong? :(

**Comment by tqchen** [comment]
@tvm-bot rerun

File: tests/python/contrib/test_hexagon/test_models.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def get_mobilenet():
     """Download and import mobilenet model with ONNX"""
     onnx = pytest.importorskip("onnx")
 
-    model_url = "https://github.com/onnx/models/raw/main/vision/classification/mobilenet/model/mobilenetv2-7.onnx"  # pylint: disable=line-too-long
+    model_url = "https://github.com/onnx/models/raw/main/validated/vision/classification/mobilenet/model/mobilenetv2-7.onnx"  # pylint: disable=line-too-long
```
Comment (yongwww) [inline tests/python/contrib/test_hexagon/test_models.py:33]: seems the `https://github.com/apache/tvm/blob/main/tests/scripts/request_hook/request_hook.py` should be updated.

Comment (shaoyuyoung) [reply tests/python/contrib/test_hexagon/test_models.py:33]: I think it can work now

**Comment by shaoyuyoung** [comment]
@leandron sry to bother you :( 

Can you take a look at this?

I'm not sure I did the right thing

**Comment by lhutton1** [comment]
Thanks for the patch @shaoyuyoung, I believe this has been superseded by #17015, so I'll close this patch. Please reopen if that's not the case.

**Pull request closed without merging**